config/splat.us.stcen.yaml

```diff
@@ -39,7 +39,9 @@ segments:
       - [0xD42C, rodata]
       - [0xD4B0, .rodata, D600]
       - [0xD4D8, .rodata, 11280] # EntityRedDoor
-      - [0xD4F0, rodata]
+      - [0xD4F0, .rodata]
```

Comment:
I just cracked this one a couple of hours ago:

```c
static const char* g_goldCollectTexts[] = {
    _S("$1"),   _S("$25"),  _S("$50"),   _S("$100"),  _S("$250"),
    _S("$400"), _S("$700"), _S("$1000"), _S("$2000"), _S("$5000"),
};
```

Reply:
For my own knowledge, do these automatically get padded to 8 bytes (the smallest alignment of the longest string) or should this be declared as `char [][8]` to ensure alignment?

Reply:
I just checked. Yes, strings seem to be aligned by 8 bytes and padded automatically.